app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -675,9 +684,11 @@ public void onSpeakingEnded() {
         case (AudioManager.AUDIOFOCUS_LOSS):
           if (!tts.currentTTSTask.paused) tts.pauseOrResume();
           pauseTTSButton.setText(R.string.tts_resume);
+          sendNotification("resume",this);
```

Comment:
Instead of using this as hard-coded ```"resume"```, you should use it from ```strings.xml```. ```Pause``` and ```Resume``` strings are already declared there.